packages/blade/src/components/Box/__tests__/useMemoizedStyles.web.test.tsx

```diff
@@ -0,0 +1,17 @@
+import { getDependencyProps } from '../BaseBox/useMemoizedStyles.web';
+
+describe('getDependencyProp', () => {
+  it('should return react usememo dependency prop', () => {
+    expect(
+      getDependencyProps({
```

Comment:
to actually test shouldn't we test the hook itself? we can render the hook and assert values right?

Reply:
That test is covered in `styled-props` test files. It's a bit more end-to-end.

We can't exactly unit test the hook. We'll have to mock a component that uses that hook and then assert the values there. styled-props tests are doing the same thing.

Reply:
added one small mock test